README.md

```diff
@@ -1,3 +1,4 @@
-# Northcoders News API
-
-For instructions, please head over to [L2C NC News](https://l2c.northcoders.com/courses/be/nc-news).
+To succesfully connect to the Database you must create 2 .env files:
+.env.test=PGDATABASE
```

Comment:
This isn't very clear. Reading this makes it seem like the file should be 

.env.test=SOMETHING - which is incorrect and would not be very safe if we were going along with that format. There should be a clear distinction between what the file name is and what the environment variable should be.

Reply:
sorry saleh i'm not fully sure how to approach this